tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -1385,8 +1424,9 @@ static void build_image(const char *output,
 
 	/* Some devices need the extra-para partition to accept the firmware */
 	if (strcasecmp(info->id, "ARCHER-C25-V1") == 0 ||
-	    strcasecmp(info->id, "ARCHER-C60-V2") == 0 ||
-	    strcasecmp(info->id, "TLWR1043NV5") == 0) {
+			strcasecmp(info->id, "ARCHER-C59-V2") == 0 ||
+			strcasecmp(info->id, "ARCHER-C60-V2") == 0 ||
+			strcasecmp(info->id, "TLWR1043NV5") == 0) {
```

Comment:
Looks like you replaced spaces with tabs here ...

Reply:
Yes, there are a few other multi-line if statements in the file which  use tabs ([L1506](https://github.com/kicken/openwrt/blob/325b390189d0139d51319d36f61c32383c75b105/tools/firmware-utils/src/tplink-safeloader.c#L1506), [L1708](https://github.com/kicken/openwrt/blob/325b390189d0139d51319d36f61c32383c75b105/tools/firmware-utils/src/tplink-safeloader.c#L1708)) so when adding the new line I changed this one to match that style.

I will revert it to spaces if preferred.

Reply:
I just realized the change compared to before. I'm not in a position to tell what's preferred or not ...

Reply:
as @blogic wrote on IRC, that's a valid point, please undo your style changes from your PR @kicken